tests/screens/LoginScreen.spec.js

```diff
@@ -34,113 +27,15 @@ describe('<LoginScreen />', () => {
       </Provider>,
     );
 
-    data = {
-      user: {
-        email: userEmail,
-        password: userPassword,
-      },
-    };
-
-    response = {
-      user: {
-        username: null,
-        password: null,
-        id: 1,
-        email: userEmail,
-      },
-    };
-
-    emailInput = wrapper.find('TextInput').at(0);
-    passwordInput = wrapper.find('TextInput').at(2);
-    submitButton = wrapper.find('Button').at(0);
+    emailInput = wrapper.find('Input').at(0);
+    passwordInput = wrapper.find('Input').at(1);
   });
 
   it('should display an email input field', () => {
-    expect(emailInput.props().name).toEqual('email');
+    expect(emailInput.props().label).toEqual('Email');
   });
 
   it('should display a password input field', () => {
-    expect(passwordInput.props().name).toEqual('password');
-  });
-
-  describe('submit Login with valid email/password inputs', () => {
```

Comment:
why are you removing all these test cases?

Reply:
I've removed most of the test cases because they were directly tied to redux-forms view hirearchy and method api (most checks used `submit`, `isValid`, etc.). 

Ideally, I should write new tests for these, but since  we are still evaluating FinalForm and Formik for the future I'm not sure it's worth 🤷‍♂

Reply:
Also, we need to review the whole test setup, because it's enzyme is still failing randomly. Maybe we could try and do something with detox, cypress style.